src/client/src/app/store/reducers/user/user.reducer.ts

```diff
@@ -35,6 +35,11 @@ export const reducer = createReducer(
     const users = [...state.users];
     users.push(action.data);
     return {...state, users}
-  })
+  }),
+
+  on(logoutUserSuccess, (state, action) => {
+    return {...state, loginUser: null}
+  }),
+  
```

Comment:
The code is working properly, but I think this file does not belong to this particular issue and pull request, it could have done in another pull request by creating new issue for handling logout.

Reply:
I agree with Aman on this part since this PR is all about cleaning up some navbar components. Good catch btw.

Reply:
Yes Aman, but i want to look at how logout work on main-nav at that time.I think I have to work on more with logout still and will do in seperate PR.